### ISSUE TITLE
Can you please repost ncc under your accont?

### ISSUE THREAD

**Issue by pfalcon** [gnuless/accs#1]
Hi, can you please repost ncc under your own account, as it was before you deleted it? Because many fork-hoarders forked it before, and it's no longer marked as a fork and people star it, as if it were a place of origin, e.g. https://github.com/moneytech/ncc

Thanks!

**Comment by gnuless** [comment]
I’ve abandoned that project.. it was a toy/proof-of-concept.  I’m not sure I have whatever sources were forked.  Sorry.

C.

**Comment by pfalcon** [comment]
> it was a toy/proof-of-concept

Yeah, I know. And that's great. People love such. They are much more cute than gcc/llvm mess.

> I’m not sure I have whatever sources were forked. 

Well, above is a link to those sources, and who would be in better position to look that no trojan was planted into them and re-uploaded to your account? ;-). Obviously, I don't pledge. It's mostly about who gets credit and about continuity of our quests - if creators abandon their cute hacks, it's always sad. Anyway, I was kinda surprised to see some "interesting" exchanges in zephyr project, followed by deletion of your account (including the ncc thingy), so just stopped by to say "Hi" and "Merry Christmas"!
